src/widget/taskBar.js

```diff
@@ -607,6 +611,21 @@ let TileableItem = GObject.registerClass(
             this.container.add_child(this.title);
             this.container.add_child(this.endIconContainer);
         }
+        
+        setStyle() {
+            this.updateTitle();
+            if(this.style == 'full') {
+                this.title.natural_width = 160
+                this.title.natural_width_set = 1;
+            } else {
+                this.title.natural_width_set = 0;
+            }
```

Comment:
And since the full text should reach the maximum width more easily I would prefer le the size at flexible since it's allow it to be compressed when many tabs are present. It's should already be less distractive. If we want we can reduce the maximum size a little in order to reach it even easier

Reply:
Tabs will shrink anyway and full text width will not reach in a lot of cases.